com.spotify.Client.json

```diff
@@ -53,7 +53,12 @@
                 {
                     "type": "archive",
                     "url": "https://download.gnome.org/sources/libnotify/0.7/libnotify-0.7.9.tar.xz",
-                    "sha256": "66c0517ed16df7af258e83208faaf5069727dfd66995c4bbc51c16954d674761"
+                    "sha256": "66c0517ed16df7af258e83208faaf5069727dfd66995c4bbc51c16954d674761",
+                    "x-checker-data": {
+		        "type": "gnome",
+			"name": "libnotify",
```

Comment:
If there's one release of libnotify a year, it's a miracle. I don't think it makes sense.

Reply:
Doesn't it make even more sense because people will forget that the releases are there?